pocket_lab/pl_gui_views.h

```diff
@@ -25,6 +25,11 @@
 /********************** Macros and Constants Definition ***********************/
 /******************************************************************************/
 
+#define PL_GUI_ADD_POWER_UP_DEF_VIEW {\
+	.view_name = " ", \
```

Comment:
Should the power up view not have any name?

Reply:
I have not given any name intentionally as this is not a functional tab, but if I get any feedback from apps, I will consider providing name